pkgs/development/python-modules/nxt-python/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchgit
+, isPy3k
+, pyusb
+, pybluez
+, pytest
+, git
+}:
+
+buildPythonPackage rec {
+  version = "3.0.1";
+  pname = "nxt-python";
+  format = "setuptools";
+
+  src = fetchgit {
+    url = "https://github.com/schodet/nxt-python.git";
+    rev = version;
+    sha256 = "004c0dr6767bjiddvp0pchcx05falhjzj33rkk03rrl0ha2nhxvz";
+    branchName= "master";
+  };
+
+  propagatedBuildInputs = [ pyusb pybluez pytest git ];
+
+  meta = with lib; {
+    description = "Python driver/interface for Lego Mindstorms NXT robot";
+    homepage = "https://github.com/schodet/nxt-python";
+    license = licenses.gpl3;
+    platforms = platforms.unix ++ platforms.darwin ++ platforms.windows;
```

Comment:
```suggestion
```
Apart from windows (which nix doesn't support) that's already default with buildPythonPackage

Reply:
Ah had no idea about the default, will remove.